### DISCUSSION TITLE
How to manually trigger init

### DISCUSSION THREAD

**Discussion by mark24slides** [tiredofit/docker-db-backup#324] [Legacy Q&A]
I've checked dockerfile, no cmd or entrypoint. Tried various ways to run 00-startup/99-container (catch only errors).

Tried to run version 4.0.34, in kubernetes, mode=MANUAL, as CronJob (sheduled job, created and destroys resources on exit, no need to have always working container in background).
If I add command: `backup-now` (even with sleep 30), container starts without prepared files and init scripts (that adds backup01...), as **it rewrites standard startup command**.

**What command can be used in postexec or command, to trigger dynamic scripts creation and config (+add `backup-now` at the end)?**

P.S. In 3.12.2 used "command: ["backup-now"], as all scripts and files were generated during image build, were already present during container startup.

P.P.S. AUTO mode works, BUT ignores MANUAL_RUN_FOREVER=FALSE, and continues to run container (kubernetes cronjob can not finish as script continues to run), even after backup was done.

**Comment by mark24slides** [comment]
Is it, also, possible to enable param. MANUAL_RUN_FOREVER=FALSE for MODE=AUTO, so after backup ops. script exited and returned 0?

**Comment by korridor** [comment]
@mark24slides Did you ever find a solution for this? I have the same problem with version 4.1.100
